### PR TITLE
Deprecate repo

### PR DESCRIPTION
**Pull request by carlossvg**
- Updates README with a deprecation message
- Updates index.md to automatically redirect to the new URL
- Removes not needed files

### CODE REVIEW

**Comment by christophebedard** [comment]
Will we get 404 errors for existing links? Seems like only the root page (i.e., https://real-time-working-group.readthedocs.io/en/latest/) will show the "This website has been deprecated" message. It would be better to have all non-root pages link to the root page to avoid 404 errors (and let people know about the switch), or have each page redirect to the corresponding page on the new website.

**Comment by carlossvg** [comment]
> Will we get 404 errors for existing links? Seems like only the root page (i.e., https://real-time-working-group.readthedocs.io/en/latest/) will show the "This website has been deprecated" message. It would be better to have all non-root pages link to the root page to avoid 404 errors (and let people know about the switch), or have each page redirect to the corresponding page on the new website.

Good point. I will do that.

**Comment by carlossvg** [comment]
@christophebedard I will try to do the trick by adding a custom 404.html redirecting to the new documentation. Like we do here but with a proper message https://github.com/ros-realtime/ros2_realtime_benchmarks/blob/main/docs/404.html.

File: README.md [hunk 1]
```diff
@@ -1,54 +1,6 @@
 # ROS 2 Real-Time Working Group Documentation
 
-[![Documentation Status](https://readthedocs.org/projects/real-time-working-group/badge/?version=rolling)](https://real-time-working-group.readthedocs.io/en/rolling/?badge=rolling)
+This repository has been deprecated.
```
Comment (LanderU) [inline README.md]:
```suggestion
## This repository has been deprecated. 
```

**Comment by LanderU** [comment]
We should add the custom 404.html yes.

**Comment by carlossvg** [comment]
@christophebedard @LanderU I used [sphinx-reredirects](https://documatt.gitlab.io/sphinx-reredirects/) to redirect all the pages. This will automatically redirect all the old URLs to the new root URL. I think this is a cleaner solution.

Edit: it is possible to redirect a specific html to the corresponding html in the new UR. I think this is a bad idea because if the structure changes, users will get a 404 error in the new URL.

**Review by LanderU — APPROVED** [review]

**Review by christophebedard — APPROVED** [review]

**Pull request merged**